src/Azure.Functions.Cli/Diagnostics/LoggerConsole.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Colors.Net;
+using Microsoft.Extensions.Logging.Console.Internal;
+using System;
+
+namespace Azure.Functions.Cli.Diagnostics
+{
+    public class LoggerConsole : IConsole
+    {
+        public void Flush()
+        {
+            
+        }
+
+        public void Write(string message, ConsoleColor? background, ConsoleColor? foreground)
+        {
+            ConsoleColor color = foreground ?? Console.ForegroundColor;
+            var value = ColorString(message, color);
+            ColoredConsole.Write(value);
+        }
+
+        public void WriteLine(string message, ConsoleColor? background, ConsoleColor? foreground)
+        {
+            Write(message + Environment.NewLine, background, foreground);
+        }
+
+        private static RichString ColorString(string value, ConsoleColor color)
+        {
```

Comment:
Disregard the last comment (which I thought I had deleted shortly after posting :/)